interop-tests/src/bin/ping.rs

```diff
@@ -32,11 +32,11 @@ async fn main() -> Result<()> {
         .unwrap_or_else(|_| "true".into())
         .parse::<bool>()?;
 
-    let test_timeout = env::var("test_timeout")
+    let test_timeout = env::var("test_timeout_seconds")
         .unwrap_or_else(|_| "10".into())
-        .parse::<usize>()?;
+        .parse::<u64>()?;
```

Comment:
changed from `usize` to `u64` because of [`Duration::from_secs`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_secs)